spec/features/user/update_article_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+feature "Update  Article" do
+  let(:user) { create :user }
+  let(:article_attributes) { attributes_for(:article) }
```

Comment:
I think that when your test should check create/update ability, than you need to set static texts and after form submit check that user will see changed fields on page.
e.g.

```
let(:article_attributes) { { title: "new_title", text: "test" } }
let(:article) { create :article, user: user, title: "hello", text: "world" }

scenario "User updates article" do
    visit edit_users_article_path(article)

    fill_form(:article, article_attributes)
    click_button "Update Article"

    expect(page).to have_content("Article was successfully updated.")
    expect(page).to have_content("new_title")
    expect(page).to have_content("test")
  end
```